modules/folder/README.md

```diff
@@ -301,6 +303,56 @@ module "folder-sink" {
 # tftest modules=5 resources=14 inventory=logging.yaml
 ```
 
+## Data Access Logs
+
+Activation of data access logs can be controlled via the `logging_data_access` variable. If the `iam_bindings_authoritative` variable is used to set a resource-level IAM policy, the data access log configuration will also be authoritative as part of the policy.
+
+This example shows how to set a non-authoritative access log configuration:
+
+```hcl
+module "folder" {
+  source = "./fabric/modules/folder"
+  parent = "folders/657104291943"
+  name   = "my-folder"
+  logging_data_access = {
+    allServices = {
+      ADMIN_READ = ["group:organization-admins@example.org"]
```

Comment:
I think we should mention somewhere that these are excluded identities. It's not clear at first glance what they mean.

Reply:
good point, addressed in the most recent commit